blueocean-personalization/src/main/js/components/DashboardCards.jsx

```diff
@@ -49,6 +53,23 @@ export class DashboardCards extends Component {
             return null;
         }
 
+        const pausedCards = this.props.favorites
+          .filter(favorite => favorite.item.latestRun.state === 'PAUSED')
+          .map(favorite => {
+              const pipeline = favorite.item;
+
+              return (
+                <div key={favorite._links.self.href}>
+                    <PipelineCard
+                      router={this.props.router}
+                      runnable={pipeline}
+                      favorite
+                      onFavoriteToggle={(isFavorite) => this._onFavoriteToggle(isFavorite, favorite)}
+                    />
+                </div>
+              );
+        });
+
         const favoriteCards = this.props.favorites.map(favorite => {
```

Comment:
@cliffmeyers as soon we have a way for that I will refactor the lot but now it makes sense as is.